coda-prototype/UIUtils.js

```diff
@@ -491,7 +490,55 @@ var UIUtils = (function() {
         isScrolledToTop: function(tableContainer) {
             return UIUtils.checkVisible($(".message-row").first()[0]);
             //return $("#message-panel").scrollTop() === 0;
-        }
+        },
+
+        /**
+         * Displays the given message in a green banner at the top of the screen.
+         * Success banners are automatically dismissed after a few seconds, and the contents automatically cleared.
+         * @param message HTML to display in the banner.
+         */
+        displayAlertAsSuccess(message) {
+            // Display the alert.
+            let successAlert = $("#alert");
+            successAlert.removeClass("alert-danger").addClass("alert-success");
```

Comment:
Where does alert-danger come from? Is that a default? Is there a way we can avoid coupling this code with the place that sets this?

Reply:
The error case.

I think I would prefer to rewrite the alerts as toasts (#83) than to implement banner alerts properly.